engine/gamesys/src/gamesys/components/comp_spine_model.cpp

```diff
@@ -167,7 +167,7 @@ namespace dmGameSystem
 
                 uintptr_t descriptor = (uintptr_t)dmGameSystemDDF::SpineAnimationDone::m_DDFDescriptor;
                 uint32_t data_size = sizeof(dmGameSystemDDF::SpineAnimationDone);
-                dmMessage::Result result = dmMessage::Post(&sender, &receiver, message_id, 0, descriptor, &message, data_size, 0);
+                dmMessage::Result result = dmMessage::Post(&sender, &receiver, message_id, 0, component->m_FunctionRef, descriptor, &message, data_size, 0);
```

Comment:
Since we moved the function ref away from the URL struct, each sender/caller needs pass it vi9a the userdata2 parameter.

That way it will be picked up by comp_script.cpp and be called accordingly.